cinema_project/base_app/jobs.py

```diff
@@ -0,0 +1,17 @@
+from .models import Reservation
+from .utils import thirty_minutes_ahead
+
+from apscheduler.schedulers.background import BackgroundScheduler
+
+
+def remove_unconfirmed_reservations() -> None:
+    Reservation.objects.filter(
+        is_confirmed=False,
+        schedule__schedule_time__lte=thirty_minutes_ahead()
+    ).delete()
```

Comment:
We should mark them as expired or invalid. We don't want to actually delete them. If something unplanned happens and a customer's reservation gets deleted, we don't have any clue / proof it ever existed.